src/test/ui/unpretty-expr-fn-arg.rs

```diff
@@ -0,0 +1,13 @@
+// Regression test for the ICE described in #82328. The pretty-printer for
+// compile-flags: -Zunpretty=hir,typed
```

Comment:
I'd prefer to see this with the other test config annotations (below `// check-pass`)

Reply:
This was intentional, so that the flag (sort of) integrates into the other text (as the ICE only happens with this specific pretty-printer, and not, say, `-Zunpretty=hir`).

Reply:
Right, I figured that was what you wanted to do. I still think it would be better for all configuration options to be together so that someone reading the code can see all of them in the same spot. Mentioning `-Zunpretty=hir,typed` in the comment is good though

Reply:
OK, fixed.